src/scripts/main.js

```diff
@@ -1,7 +1,26 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const div = document.createElement('div');
+  const body = document.querySelector('body');
+
+  div.className = `notification ${type}`;
+
+  div.insertAdjacentHTML("afterbegin",
+    `<h2 class="title">${title}</h2>
```

Comment:
```suggestion
  div.insertAdjacentHTML('afterbegin',
    `<h2 class='title'>${title}</h2>
```
In HTML we use double-quotes but in JS strings must use singlequote